landbosse/tests/model/test_SubstationCost.py

```diff
@@ -10,7 +10,11 @@ class TestSubstationCost(TestCase):
     def setUp(self):
```

Comment:
Please remove chances to the test data before you merge.

Reply:
Hi @eberlea, just so I'm on the same page, would you like me to revert code in the test_SubstationCost.py to a version that sits in the develop branch?

Reply:
Ah, if the changes are in the develop branch, then just leave them be. If you're making the changes in your branch, then please revert.